### PR TITLE
fix: nav and footer logo

### PR DESCRIPTION
**Pull request by isaackps**
## In this PR:
- Update nav logo size
- Update footer logo size

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -35,6 +35,8 @@ const siteConfig = {
         alt: "TradeTrust Logo",
         src: "img/logo/logo-tt-full.svg",
         srcDark: "img/logo/logo-tt-full.svg",
+        width: 168,
+        height: 50,
```
Comment (simboonlong) [inline docusaurus.config.js:39]: think these does nth as they are overwritten by base `img` styles, can remove

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "@docusaurus/core": "^2.0.0-beta.20",
     "@docusaurus/preset-classic": "^2.0.0-beta.20",
-    "@govtechsg/tradetrust-ui-components": "^2.18.0",
+    "@govtechsg/tradetrust-ui-components": "^2.18.6",
     "@govtechsg/tradetrust-utils": "^1.2.1",
```
Comment (simboonlong) [inline package.json:17]: `2.18.7`?

File: src/css/styles.css [hunk 3]
```diff
@@ -130,6 +130,12 @@
   @apply text-gray-400;
 }
 
+.navbar__logo img {
+  @apply w-[168px];
+  @apply h-10;
+  @apply -mt-2;
+}
```
Comment (simboonlong) [inline src/css/styles.css:137]: estimating of alignment + size setting might not be ideal here

File: docusaurus.config.js (hunk 1 above)
Comment (isaackps) [reply docusaurus.config.js:39]: ok

File: package.json (hunk 2 above)
Comment (isaackps) [reply package.json:17]: yup, cos docusaurus not using our nav bar so it wont really affect it.

File: src/css/styles.css (hunk 3 above)
Comment (isaackps) [reply src/css/styles.css:137]: agree, but there are no place for that in the component, since we are using docusaurus component.

Comment (simboonlong) [reply src/css/styles.css:137]: the logo (red outline) content is extending out of its parent box model (blue outline). these are usually the cause for broken layouts

Screenshot 2022-06-01 at 9 46 30 AM

thoughts: if there is a compulsory requirement for a fixed dimensions logo, try applying only the width to the parent box instead (leave the height), then logo max-width: 100%. if not compulsory, even better. leave it as it is

Comment (isaackps) [reply src/css/styles.css:137]: hmm ok will fix it. was asked to increase the size to the one in figma so need to adjust it.

**Review by simboonlong — APPROVED** [review]

**Pull request merged**